### PR TITLE
Dockerfile for alpine trace agent

### PR DESCRIPTION
**Pull request by archerabi**

### CODE REVIEW

**Comment by archerabi** [comment]
@imsky @alikhan-io do you want me to add a .pull-review to this repo ? 
I don't see CI for baseimage, is it built 'by hand' ?

**Comment by imsky** [comment]
you'll need to add CI via DockerHub

up to you about .pull-review - in general i think we should use it, but not a requirement

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,49 @@
+FROM better/baseimage:alpine-3.5-r1
```
Comment (imsky) [inline Dockerfile]: please use alpine 3.7 base image

**Comment by imsky** [comment]
does this replace the agent, btw, or is it going to be run separately?

**Comment by archerabi** [comment]
@imsky so okay to put this in the public repo ? there isn't anything specific to better, but just wanted to check. 
The trace agent is a separate component, it runs by itself in this image.

**Comment by imsky** [comment]
yes okay to add to public repo, we have several public images: https://hub.docker.com/r/better/

File: Dockerfile (hunk 1 above)
Comment (archerabi) [reply Dockerfile]: Done

**Comment by archerabi** [comment]
@imsky i think i need permissions to push to better/*. my username is archerabi

**Comment by imsky** [comment]
credentials should be in LastPass for our common DockerHub account, let me know if you're not able to access

**Comment by archerabi** [comment]
Done https://hub.docker.com/r/better/dd-trace-agent-alpine/
Thanks @imsky

**Comment by imsky** [comment]
:+1:

**Comment by archerabi** [comment]
Thanks @imsky

**Pull request merged**